docs/code/getting-started.md

```diff
@@ -675,4 +675,141 @@ Let’s recap what we learned in this section:
+                            <setting name="TokenPath">/o/oauth2/token</setting>
+                            <setting type="int" name="UiPolicy">0</setting>
+                            <setting name="RedirectUri">http://elementaryos.org</setting>
+                            <setting name="ResponseType">code</setting>
+                            <setting name="Scope">https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email https://www.google.com/m8/feeds/</setting>
+                            <setting name="AuthQuery">access_type=offline&amp;approval_prompt=force</setting>
+                            <setting name="ClientId">generic-id.apps.googleusercontent.com</setting>
+                            <setting name="ClientSecret">generic-secret</setting>
+                            <setting type="as" name="AllowedSchemes">['https','http']</setting>
+                            <setting type="as" name="AllowedRealms">['google.com']</setting>
+                            <setting type="as" name="Realms">['accounts.google.com']</setting>
+                        </group>
+                    </group>
+                </group>
+            </template>
+        </service>
+
+Everything is pretty straightforward, but here are some precisions to .service files:
+
+* The `type` should only be generic if you’re sure that it would be handled by any application implementing generic services (e-mail is the most famous one)
+* The `icon` should be a generic icon described by the [freedesktop.org specification](http://standards.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html) when possible or should be provided along with the service package
+* The `translations` is the domain that should be installed and provide translations for the description
+* The .service file inherit from all the fields provided by the provider. The `template` addition above should only be present if you’re requesting other privileges than the .provider already provides (here we want to access to other google Scopes).
+
+## Declare your Application {#poa-declare-application}
+
+Each application need their .application file in "/usr/share/accounts/applications/"
+The file should look as above:
+
+        <?xml version="1.0" encoding="UTF-8"?>
+        <application id="my-address-book">
+            <description>My Address Book</description>
+            <desktop-entry>my-address-book.desktop</desktop-entry>
+            <translations>my-address-book</translations>
+            <services>
+                <service id="google-contacts">
+                </service>
+            </services>
+            <service-types>
+                <service-type id="contacts">
+                </service-type>
+            </service-types>
+        </application>
+
+In some cases, your application will handle all kind of services (such as e-mails). If so, you can just omit the `services` and just fill the `service-types` you want.
+
```

Comment:
This seems really incomplete. I've created these files, now what? How does my app actually interact with the data that these things provide?